lib/tasks/foreman_ansible_tasks.rake

```diff
@@ -3,10 +3,11 @@
 # Tests
 namespace :test do
   desc 'Foreman Ansible plugin tests'
-  Rake::TestTask.new(:foreman_ansible) do |t|
-    test_dir = File.join(File.dirname(__FILE__), '../..', 'test')
+  Rake::TestTask.new(:foreman_ansible => 'db:test:prepare') do |t|
```

Comment:
I wonder why this dependency is needed, given other plugins don't do it. I think we can simply drop it instead.